Assets/Scripts/PlayerMovement.cs

```diff
@@ -0,0 +1,39 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class PlayerMovement : MonoBehaviour {
+    private Rigidbody2D rgBody;
+    [SerializeField]
+    private float movementSpeed;
+
+    [SerializeField]
+    private Transform[] groundedPoints;
+
+	void Start ()
+    {
+        rgBody = GetComponent<Rigidbody2D>();
+	}
+	
+	void FixedUpdate ()
+    {
+        float horizontal = Input.GetAxis("Horizontal");
+        Debug.Log(horizontal);
+
+        movementHandler(horizontal);
+	}
+
+    private void movementHandler(float horizontal)
+    {
+        rgBody.velocity = new Vector2(horizontal * movementSpeed, rgBody.velocity.y);
+
+        shouldJump(horizontal);
```

Comment:
in general, methods starting with `should`/`is`/`can` etc. return booleans. So what I meant was to put the stuff inside the `if()` in a method, and leave the rest in the movementHandler.